vignettes/data.Rmd

```diff
@@ -18,16 +18,18 @@ knitr::opts_chunk$set(
 
 The model requires a data table containing the following columns:
 
-stan_id:
-date:
-obs_id:
-t_since_last_exp:
-titre:,
-titre_type: ,
-titre_type_num: ,
-last_exp_date: ,
-relevant_last_exp_date: ,
-censored:
+### pid
+A unique numeric id to identify a person across observations
+### date
+The date of the observation
+### last_exp_date
+The last date on which the person was exposed
+### titre_type
+The name of the titre
+### value
+The value of the titre
+### censored
+Whether this observation should be censored: -1 for lower, 1 for upper, 0 for none
```

Comment:
In the spirit of moving more data processing into the package, we should probably actually take censoring limits as arguments, rather than requiring this as a column in the input data?

Reply:
Yeah, this sounds like a great idea. I think it would be relatively common to have censored data below and/or above but not necessarily both. It will also be relatively common to have data which just isn't censored at all. So I think being flexible around this is definitely the right approach. Allowing the users to specify their own censored limits is certaintly a good idea, given that different assays are calibrated differently and not necessarily standardised.